### PR TITLE
feat: remove argparse option

### PR DESCRIPTION
**Pull request by esciara**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -116,8 +116,7 @@ set up separately):
   * Publish the release on the chosen hosting (GitLab or GitHub, cloud or self-hosted).
   * Publish the released package to the chosen Python package repository.
 
-* Command line interface (optional), with testing, using Cleo_ (default), Click_ or
-  Argparse_.
+* Command line interface (optional), with testing, using Cleo_ (default), Click_.
```
Comment (slequeux) [inline README.rst]: Replace the last comma with `or` ?

**Review by slequeux — COMMENTED** [review]
Just a minor feedback.
This looks fine

File: README.rst (hunk 1 above)
Comment (slequeux) [reply README.rst]: @esciara There was only this very minor comment before being able to close this PR.
Could you check on this ?

Comment (esciara) [reply README.rst]: done

**Pull request merged**